prow/config.yaml

```diff
@@ -241,7 +241,7 @@ presubmits:
         - name: ssh
           secret:
             secretName: ssh-key-secret
-            defaultMode: 0400
+            defaultMode: 400
```

Comment:
what's going on here? some formatting thing you have?

Reply:
Nothing I intentionally set up. I installed bazel and ran `bazel run //jobs:config_sort` as the test failure suggested and then committed the changes that it made.